packages/db/fauna/schema.graphql

```diff
@@ -195,43 +196,74 @@ type Upload {
 }
 
 """
-Filecoin deal for a pin.
+A batch of content for inclusion in a Filecoin deal.
 """
-type Deal {
+type Batch {
```

Comment:
We could track when a Batch is created so we can periodically delete batches that got dumped in favour of others and will never make it on chain.

Reply:
Note - I do not intend to send dealinfo up until there's an actual on-chain publish (prior to seal). But still worthwhile to do for the future.